File: Tarea2_LeyvaBanda/Tarea2.cs
using System;

class Carro {
    public string Marca;
    public string Modelo;
    public int Año;
}

class Program {
    static void Main() {
        Carro c = new Carro();
        c.Marca = "Nissan";
        c.Modelo = "Versa";
        c.Año = 2020;

        Console.WriteLine($"Carro: {c.Marca} {c.Modelo} ({c.Año})");
    }
}

File: Tarea2_LeyvaBanda/Tarea2.py
class Carro:
    def __init__(self, marca, modelo, año):
        self.marca = marca
        self.modelo = modelo
        self.año = año

c = Carro("Nissan", "Versa", 2020)
print(f"Carro: {c.marca} {c.modelo} ({c.año})")
